docs/feature-set.md

```diff
@@ -0,0 +1,105 @@
+PC Engines apu feature set
+==========================
+
+## Coreboot features
```

Comment:
@miczyg1 let's try to follow https://www.coreboot.org/Board:pcengines/apu2 by dividing this document between platforms (one one md or multilple md). So we should have device/functionality, status, comment and test case. Our target is to have the same styling as coreboot feature list but one more column with a test case.

So this page should be divided between platforms apu1/2/3/4/5 and in future alix.